docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How we improved Gatsby's accessibility in v2 by switching to @reach/router
+date: 2018-09-24
+author: Amberley Romo
+tags: ["routing", "v2", "accessibility"]
+---
+
+We recently announced the [second major release of Gatsby](/blog/2018-09-17-gatsby-v2/) 🚀. One change we want to highlight is the switch to using [@reach/router](https://reach.tech/router) to improve the accessibility of routing in Gatsby sites. What is @reach/router, and what are the benefits of undertaking the switch?
+
+## What is @reach/router?
+
+@reach/router is a routing library for React written and maintained by the founder of react-router, [Ryan Florence](https://twitter.com/ryanflorence). After [investigating the tradeoffs](https://github.com/gatsbyjs/gatsby/issues/5656), we made the leap to depending on @reach/router:
+
+- [Accessibility](https://reach.tech/router/accessibility) is a first-class concern
+- Smaller bundle size (25% decrease, 8kb to 6kb gzipped)
```

Comment:
I thought we dropped more here - we replaced both `react-router` (8kb) and `react-router-dom` (10kb)

Reply:
Check out the https://bundlephobia.com/ sizes. Her numbers are post-gzip.

Reply:
So it's 18.4 kb gzipped vs. 6, correct?

Reply:
10.4kb (react-router-dom)
8kb (react-router)

vs. 

6kb (@reach/router)

I'll update my presentation(s) with that too.

Reply:
@KyleAMathews I can include the change to the v2 release post in this PR. I'm going to go with what @DSchau said above, unless i hear otherwise? 👍

Reply:
It's reasonable 🤷‍♂️ Technically we're doubling up on a few as @stefanprobst noted.